src/bridge/bridge.c

```diff
@@ -467,7 +436,7 @@ run_bridge (const gchar *interactive,
   gboolean closed = FALSE;
   const gchar *directory;
   struct passwd *pwd;
-  GPid daemon_pid = 0;
+  g_autoptr (GSubprocess) dbus_daemon_process = NULL;
   GPid agent_pid = 0;
```

Comment:
FTR: once this gets a +1 and lands, I'm happy to rewrite the SSH agent setup as well, as right now it's asymmetric. But I wanted to get one right first.